lib/blocks.php

```diff
@@ -372,3 +372,30 @@ function gutenberg_register_legacy_social_link_blocks() {
 }
 
 add_action( 'init', 'gutenberg_register_legacy_social_link_blocks' );
+
+/**
+ * Migrate the legacy `sync_status` meta key to the new `wp_pattern_sync_status` meta key.
+ *
+ * @since 16.1.1
```

Comment:
Is this a standard. Never knew that we had to add `@since` for Gutenberg-only functions

Reply:
I don't think we do by default, but seems to make sense for something like this - let's make it a standard :-)